### PR TITLE
add some leeway with spaces

### PR DESCRIPTION
**Pull request by josethirdchannel**
https://thirdchannel.atlassian.net/browse/TC-4452 
https://thirdchannel.atlassian.net/browse/TC-4479

### CODE REVIEW

File: js/apps/shared/utils/handlebarsHelpers.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ define(function (require) {
     });
 
     Handlebars.registerHelper("highlight", function (text, mentionedUsers, currentUserId, highlightWords, options) {
-        var highlightMatcher = new RegExp(/@(\w+\s{1,3}\w+)(?:\sAgents)?\s\[[^\[\t\n\r\]]+\]/g); // allow for 1-3 spaces between first, last
+        var highlightMatcher = new RegExp(/@(\w+\s{1,3}\w+(?:\sAgents)?\s{1,3})\[[^\[\t\n\r\]]+\]/g); // allow for 1-3 spaces between first, last
```
Comment (eliottwiener-thirdchannel) [inline js/apps/shared/utils/handlebarsHelpers.js]: Why did you change the matching group to include the "Agents" part?

Comment (josethirdchannel) [inline js/apps/shared/utils/handlebarsHelpers.js]: I added it to cover the case where agents have spaces at the end of their last name. I needed to include the optional (?:Agents)? so I could have a variable number of space characters between between Agent Name and [Role].

**Comment by eliottwiener-thirdchannel** [comment]
:+1:
